### PR TITLE
Modified function that determines number of avilable spots

### PR DESCRIPTION
**Pull request by rouoba**
Issue #14
Modified countAvailableSpots function

### CODE REVIEW

**Review by lsalazar1 — APPROVED** [review]

File: python/ParkingLot.py [hunk 1]
```diff
@@ -31,15 +31,25 @@ def countSensors(self):
...
+    #     for document in col:
+    #         if document["isVacant"] == False:
+    #             spotsTaken += 1
+        
+    #     return self.totalSpots - spotsTaken
+
     def countAvailableSpots(self):
-        spotsTaken = 0
-        col = list(self.collection.find())
+        availableSpots = 0
```
Comment (mbghazi) [inline python/ParkingLot.py:45]: Is it possible to have const variables in python? Otherwise i have no issues

**Review by mbghazi — APPROVED** [review]

File: python/ParkingLot.py (hunk 1 above)
Comment (rouoba) [reply python/ParkingLot.py:45]: ok. i believe our only constant variables so far are the total number of spots for each parking lot.

Comment (lsalazar1) [reply python/ParkingLot.py:45]: Yes, there's a way to have constant variables in Python. You have to import a module called "Constant". What do you want to make constant?

**Review by mariahfont — COMMENTED** [review]
Is this still making multiple documents or have we fixed that issue already?

**Comment by lsalazar1** [comment]
> Is this still making multiple documents or have we fixed that issue already?

Yeah we fixed that on the one PR you approved a couple days ago

**Review by mariahfont — APPROVED** [review]

**Review by gazie91 — APPROVED** [review]

**Pull request merged**